superset-frontend/src/visualizations/FilterBox/FilterBox.test.jsx

```diff
@@ -46,9 +47,11 @@ describe('FilterBox', () => {
           },
         ]}
         origSelectedValues={{}}
+        theme={supersetTheme}
```

Comment:
Can we use `styledShallow` instead of importing the theme?

Reply:
Hey @michael-s-molina thanks for the suggestion. I had to use `styledMount` instead. I updated the PR. Thank you!